manifests/p/PPSSPPTeam/PPSSPP/1.15.3/PPSSPPTeam.PPSSPP.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: PPSSPPTeam.PPSSPP
+PackageVersion: 1.15.3
+PackageLocale: en-US
+Publisher: PPSSPP Team
+PackageName: PPSSPP
+License: GPL-2.0
+ShortDescription: PPSSPP can run your PSP games on your PC in full HD resolution; it can even upscale textures that would otherwise be too blurry as they were made for the small screen of the original PSP

```

Comment:
```suggestion
ShortDescription: PPSSPP can run your PSP games on your PC in full HD resolution; it can even upscale textures that would otherwise be too blurry as they were made for the small screen of the original PSP
Tags:
- emulator
- psp
ReleaseNotesUrl: https://github.com/hrydgard/ppsspp#whats-new-in-1153
Documentations:
  - DocumentLabel: Wiki
    DocumentUrl: https://github.com/hrydgard/ppsspp/wiki
  - DocumentLabel: Documentation & Help
    DocumentUrl: https://www.ppsspp.org/docs/intro
```
Additional metadata